contracts/controller/src/admin.rs

```diff
@@ -118,7 +118,7 @@ impl Contract {
     pub fn get_liquidation_incentive(&self) -> Ratio {
         // TODO: Move this kind of getter that don't require admin rights somewhere else
         // incentive % + 100 %
-        self.liquidation_incentive + Ratio(10000)
+        self.liquidation_incentive + Ratio((1.0 * RATIO_DECIMALS.0 as f64) as u128)
```

Comment:
@dmytro-shash this conversion may fail to produce the number you expect (because of how float point is represented)

let's rewrite this in the following way
self.liquidation_incentive + Ratio(1 * RATIO_DECIMALS);
...
liquidation_incentive: Ratio(5 * RATIO_DECIMALS / 100)
health_threshold: Ratio(150 * RATIO_DECIMALS / 100)